src/components/Molecules/task/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import Checkbox from "../../Atoms/Checkbox/index";
+import EditButton from "../../Atoms/EditButton/index";
+import Input from "../../Atoms/Input/index";
```

Comment:
```suggestion
import Checkbox from "../../Atoms/Checkbox";
import EditButton from "../../Atoms/EditButton";
import Input from "../../Atoms/Input";
```
もしくは、`../../Atoms/Checkbox/index`というファイルは存在しない&`index.js`だけでなくCheckbox自体をimportしたいので、このように変更するとかですかね...？